channeldb/payments.go

```diff
@@ -80,6 +114,8 @@ func (ps PaymentStatus) String() string {
 // OutgoingPayment represents a successful payment between the daemon and a
 // remote node. Details such as the total fee paid, and the time of the payment
 // are stored.
+//
+// NOTE: Deprecated. Kept around for migration purposes.
```

Comment:
Can this be moved close to the migration code? I actually think we should always do one file per migration. Then this can be moved into it.

Reply:
Moved to `legacy_serialization.go`: https://github.com/lightningnetwork/lnd/pull/2761/commits/4f6c127af2c471c5f83792bafb75f9d3244939b2